### PR TITLE
[ADD] account_payment_register_keep_amount

### PR DESCRIPTION
**Pull request by SirTakobi**

### CODE REVIEW

**Review by patrickt-oforce — APPROVED** [review]

**Review by matteoopenf — APPROVED** [review]

**Comment by matteoopenf** [comment]
@pedrobaeza  is it possibile to merge?
thank you so much

File: account_payment_register_keep_amount/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#  Copyright 2023 Simone Rubino - TAKOBI
+#  License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Register Payment keep amount",
+    "summary": "Keep set amount during Payment registration.",
+    "version": "14.0.1.0.0",
+    "category": "Account-payment",
+    "website": "https://github.com/OCA/account-payment"
```
Comment (pedrobaeza) [inline account_payment_register_keep_amount/__manifest__.py:9]: Website should be the base one.

File: account_payment_register_keep_amount/readme/DESCRIPTION.rst [hunk 2]
```diff
@@ -0,0 +1,3 @@
+Allow to keep the set amount in Payment registration wizard when Journal is changed.
```
Comment (pedrobaeza) [inline account_payment_register_keep_amount/readme/DESCRIPTION.rst:1]:
```suggestion
Allow to keep the amount set in the payment registration wizard when the journal is changed.
```

**Comment by matteoopenf** [comment]
@SirAionTech @SimoRubi  can you do the fix?
thank you

**Comment by SirAionTech** [comment]
> @SirAionTech @SimoRubi can you do the fix?
thank you

I won't be working on this module in the near future; if there is work to do anybody can supersede this pull request and do it.
Please just keep the authorship of the work that has been done

**Comment by pedrobaeza** [comment]
OK, my comments are not critical, so let's merge it:

/ocabot merge nobump

**Pull request merged**